docs/integreatly/modules/ROOT/pages/js-client-offline.adoc

```diff
@@ -265,13 +263,13 @@ client.mutate<Task>(options);
 
 === Using cache update helpers for subscriptions
 
-The @aerogear/voyager-server module provides a subscription helper which can generate the necessary options to be used with Apollo Client's `subscribeToMore` function.
+The @aerogear/voyager-client module provides a subscription helper which can generate the necessary options to be used with Apollo Client's `subscribeToMore` function.
```

Comment:
lol, I'm wondering how did this change end up in this PR

Reply:
There is some strange git stuff happening here.